lib/index.js

```diff
@@ -68,23 +61,21 @@ export default [
         rank: 1,
       });
 
-      const widget = new ExampleWidget();
-      var n = document.createElement('div');
-      n.innerHTML = `
-                        <div style="background: white; padding: 1em; margin: 1em; justify-content: center;">
-                         ${slackIconStr}
-                          <br>
-                           <a href="https://www.slack.com">
-                                  <button class="slk slkInp"> Visit Sirepo Slack </button>
-                                </a>
-                        </div>
-                      `;
-      widget.node.appendChild(n);
-      widget.id = 'simple-widget-example';
-      console.log('widget.node: ', widget.node);
-      widget.node.classList.add('slack-logo');
-      shell.add(widget, 'left', {rank: 4});
-
+      var widget = new SlackWidget();
```

Comment:
Why change to var?

Reply:
I wasn't very familiar with using const for objects in JS. After reading about it I've changed it back. 42fb1f3652efd9e334cd70ed465d4ccf4db14a6e